agent/util-scripts/pbench-tool-trigger

```diff
@@ -39,31 +49,46 @@ while (<TRIG>) {
 	}
 	chomp;
 	($group, $start, $stop) = split(/:/);
-	printf "[$script]found trigger: group:$group start-trigger:$start stop-trigger:$stop\n";
-	$start_t{$group} = $start;
-	$started{$group} = 0;
-	$stop_t{$group} = $stop;
+	if ($group ne $tool_group) {
+		next;
+	}
+	if ($start eq "" || $stop eq "") {
```

Comment:
Do we need to check the trigger strings for colons? Or document a limitation?

Reply:
Posted an updated to catch it when registering.